pkg/render/apiserver.go

```diff
@@ -1258,26 +1261,31 @@ func (c *apiServerComponent) tigeraCustomResourcesClusterRole() *rbacv1.ClusterR
 				"patch",
 			},
 		},
+		{
```

Comment:
Please if you could add a comment as to why we need this permission, it will help us track over time why we added permissions and also make it easier to remove unused permissions.